templates/zerver/help/zulip-on-windows.md

```diff
@@ -0,0 +1,44 @@
+# Using Zulip on Windows
+
+Zulip has a Windows client, compatible with Windows versions 7, 8 and 10 and is identical in
+functionality to the Zulip Web Application. The client
+can be download through https://zulip.org/.
```

Comment:
You should make `https://zulip.org/` into a hyperlink for other users to click on.